src/main/java/apprenticeship/UserInput.java

```diff
@@ -0,0 +1,23 @@
+package apprenticeship;
+
+import java.util.Scanner;
+
+public class UserInput {
+    private Scanner scanner;
+    private UserInputValidator validator = new UserInputValidator();
+
+    public UserInput(Scanner s) {
+        this.scanner = s;
+    }
+
+    public int obtainValidCellSelection() {
+        System.out.println("Enter your cell selection (1 - 9): ");
+        int cell = this.scanner.nextInt();
```

Comment:
You don't need to do this right now, but in the future you might want to think about creating a JSON or YAML file that stores all of your static messages. That way, the messages are organized in one place rather than scattered throughout the codebase.